### DISCUSSION TITLE
ジャンパのハンダ付け

### DISCUSSION THREAD

**Discussion by naopolo** [Yowkees/keyball#127] [Q&A] [answered]
うまくハンダが乗らず、ハンダを取り除いたり、アルコールでふいたりを繰り返した結果、写真のような状態となってしまいました。
パターン剥がれや基盤の損傷を危惧しているのですが、故障確認、復旧の手立てはありますでしょうか？

![Uploading 3203787A-FB6C-4746-BA1B-EE4C04AE8A61.jpeg…]()

**Comment by naopolo** [comment]
写真をアップしなおしました。

![0D365600-6605-40FC-B92B-B9D27CB43BC6](https://user-images.githubusercontent.com/7826130/177256652-a645c97b-b3ac-4293-b594-e192080ff076.jpeg)

**Reply by naopolo** [reply]
この状態で、ハンダはまったくのりません。

**Reply by naopolo** [reply]
連投すいません。いろいろと試みているものの当該箇所の復旧は難しそうかなと思っています。
＃被害がほかの部分にも及びそう
そこで、可能であれば当該ジャンパがどことどこのピンをつなぐためのものか教えていただくことは可能でしょうか？
それをもとに、自分で同線などでつなぐことを試みようと思います。
基板は、トラックボールを装着しない方のものになります。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

まず、4Pピンソケットとタクトスイッチをビルドガイドの通りに半田付けしていただいて、その後下記の赤点の2個所を導線でショートしてください。これでOLEDが復活するはずです。
頑張ってください！それでも問題ありましたらご連絡ください。
![Keyball61catchup](https://user-images.githubusercontent.com/84209772/177438836-2415e770-a8ef-47d4-9262-b787f565af34.jpg)

**Reply by naopolo** [reply]
コメントいただきありがとうございます。
自分でも配線を少し調べてみて、GNDにつながるのかな？と考え、ProMicroのソケット側のGNDにつなげて動作させることができました。
RESETピンへの配線ということは、仮説があっていたのかなと推察しております。
また、何かありましたらご相談させてください。

**Reply by Yowkees** [reply]
お調べいただいた通りで、GNDです。どこのGNDでも大丈夫です🙏
